src/parser/core/healingEfficiency/HealingEfficiencyTracker.js

```diff
@@ -44,6 +44,10 @@ class HealingEfficiencyTracker extends Analyzer {
     spellInfo.damageDone = ability.damageEffective || 0;
     spellInfo.damageAbsorbed = ability.damageAbsorbed || 0;
 
+    spellInfo.manaSpent = this.manaTracker.spendersObj[spellId] ? this.manaTracker.spendersObj[spellId].spent : 0;
+    spellInfo.manaPercentSpent = spellInfo.manaSpent / this.manaTracker.spent;
```

Comment:
this line needs to stay where it is, after getCustomSpellStats is called, otherwise mana percentages will be wrong for spells where you changed their expenditure

Reply:
Good catch.